src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -725,17 +744,19 @@ struct jit_uni_eltwise_generic : public jit_uni_eltwise_kernel, public jit_gener
                     OPENVINO_THROW("unknown src_prc");
             }
 
-            switch (dst_prc) {
-                case Precision::FP32:
-                    if (!src_prc.is_float())
-                        uni_vcvtdq2ps(vmm_src, vmm_src);
-                    break;
-                case Precision::I32:
-                    if (src_prc.is_float())
-                        uni_vcvtps2dq(vmm_src, vmm_src);
-                    break;
-                default:
-                    OPENVINO_THROW("unknown dst_prc");
+            if (src_prc != dst_prc) {
```

Comment:
Even with this change i8/u8 input values are converted to I32 precision. Is it expected behavior?

Reply:
fixed, thanks!